themes/default/content/blog/pulumi-release-notes-m59/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "July 28 releases: K8s GitOps, autonaming in Google Native"
+date: 2021-07-28T08:00:00-07:00
+draft: false
+meta_desc: "Kubernetes is faster, K8s operator supports GitOps, new resources and functionality for Azure and Google Cloud, and easier getting started in the Pulumi Service"
+meta_image: meta.png
+authors:
+    - alex-mullans
+tags:
+    - features
+---
+
+Another 3 weeks have passed, so it's time for another edition of the Pulumi release notes! This iteration we've been busy smashing bugs across our products (over 100!), but we've still got some great new features to share with you as well:
+
+- New and updated cloud providers
+  - [Kubernetes operator supports GitOps workflow]({{< relref "/blog/pulumi-release-notes-m59#kubernetes-operator-supports-gitops-workflow" >}})
+  - [Kubernetes improved performance]({{< relref "/blog/pulumi-release-notes-m59#kubernetes-improved-performance" >}})
+  - [12 new resources in the Azure Native provider]({{< relref "/blog/pulumi-release-notes-m59#12-new-resources-in-the-azure-native-provider" >}})
+  - [Google Native: autonaming; easier management of project and region settings]({{< relref "/blog/pulumi-release-notes-m59#google-native-autonaming-easier-management-of-project-and-region-settings" >}})
+- Pulumi CLI and core technologies
+  - [Automation API: Select your backend]({{< relref "/blog/pulumi-release-notes-m59#automation-api-select-your-backend" >}})
+- Pulumi Service and Pulumi.com
+  - [Get started with Pulumi, directly from the Pulumi Service]({{< relref "/blog/pulumi-release-notes-m59#get-started-with-pulumi-directly-from-the-pulumi-service" >}})
+
+<!--more-->
+
+## New and updated cloud providers
+
+### Kubernetes operator supports GitOps workflow
+
+For some Kubernetes workflows, developers want to be able to store their configuration in a Git repository but ensure that their cluster automatically stays up-to-date with the latest configuration when it's checked in. With the latest update to the Pulumi Kubernetes Operator, you can set up this type of GitOps workflow. To get started, TODO
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi-kubernetes-operator/issues/50)
+
+### Kubernetes improved performance
+
+TODO
```

Comment:
@lblackstone on Monday, can you help me draft content for this item (and the other TODO a few lines up)?

Reply:
@infin8x The informer changes in this release aren't performance fixes; we're planning to work on that further later. The current changes make the provider more resilient to network flakiness, especially under heavy load. It's more of a correctness fix that most users probably won't notice. If you like, I can draft something up about it, or we can drop this section from the blog.

Reply:
Will drop this section, thanks for letting me know!